code/datums/mind.dm

```diff
@@ -1553,6 +1553,15 @@
 	spell_list += S
 	S.action.Grant(current)
 
+//To remove a specific spell from a mind
+/datum/mind/proc/remove_spell(var/obj/effect/proc_holder/spell/spell)

```

Comment:
Could you change these share same case while you're at it ? AddSpell/RemoveSpell or add_spell/remove_spell